care/facility/api/viewsets/patient_consultation.py

```diff
@@ -237,3 +238,19 @@ def patient_from_asset(self, request):
         if not consultation:
             raise NotFound({"detail": "No consultation found for this asset"})
         return Response(PatientConsultationIDSerializer(consultation).data)
+
+
+def dev_preview_discharge_summary(request, consultation_id):
+    """
+    This is a dev only view to preview the discharge summary template
+    """
+    consultation = (
+        PatientConsultation.objects.select_related("patient")
+        .order_by("-id")
+        .filter(external_id=consultation_id)
+        .first()
+    )
+    if not consultation:
+        raise NotFound({"detail": "No consultation found for this asset"})
```

Comment:
```suggestion
        raise NotFound({"detail": "Consultation not found"})
```
Let's correct this error message. 😅